db/warehouse/migrate/20241110005806_revert_warehouse_indexes_phase_1.rb

```diff
@@ -0,0 +1,103 @@
+# The 20241010005805_prune_warehouse_indexes_phase_1 ran on staging and removed some indexes that
+# proved to be in use.
+class RevertWarehouseIndexesPhase1 < ActiveRecord::Migration[7.0]
```

Comment:
for consideration, should we restore these indexes to ensure consistency between staging/prod?

Reply:
I'm definitely of two minds on this, but I think we should restore them.